src/server.h

```diff
@@ -141,14 +145,14 @@ class CServerSlots : public CServerSlots<slotId - 1>
         CreateAndSendJitBufMessage ( slotId - 1, iNNumFra );
     }
 
-protected:
-    virtual void SendProtMessage ( int              iChID,
-                                   CVector<uint8_t> vecMessage ) = 0;
+  protected:
+    virtual void SendProtMessage ( int iChID, CVector<uint8_t> vecMessage ) = 0;
 
     virtual void CreateAndSendChanListForThisChan ( const int iCurChanID ) = 0;
 
-    virtual void CreateAndSendChatTextForAllConChannels ( const int      iCurChanID,
-                                                          const QString& strChatText ) = 0;
+    virtual void
```

Comment:
Need to disallow this kind of split.  Split after the `(` after the method name.

Reply:
we can avoid this by increasing `PenaltyReturnTypeOnItsOwnLine`
I just did that and it makes a few changes:
https://github.com/jamulussoftware/jamulus/pull/1127/commits/d4dd534805b5f0770c6d17257e5a6fa079b60424